aries_cloudagent/messaging/credential_definitions/tests/test_routes.py

```diff
@@ -44,13 +41,13 @@ def setUp(self):
         )
         self.context.injector.bind_instance(BaseStorage, self.storage)
 
-        self.app = {
-            "request_context": self.context,
+        self.request_dict = {
+            "context": self.context,
         }
 
     async def test_send_credential_definition(self):
         mock_request = async_mock.MagicMock(
-            app=self.app,
+            __getitem__=async_mock.Mock(side_effect=self.request_dict.__getitem__),
```

Comment:
I couldn't find a better approach to `__getitem__` to set dict values on the `Request` object. Setting them with `request["context"]=self.context` doesn't work and setting them with `context=self.context` also doesn't work. MagicMock doesn't play well with items I think.